cmd/kubeadm/app/cmd/cmd.go

```diff
@@ -62,8 +62,6 @@ func NewKubeadmCommand(f *cmdutil.Factory, in io.Reader, out, err io.Writer) *co
 
 		`),
 	}
-	// TODO(phase2+) figure out how to avoid running as root
```

Comment:
This should be left as-is, since we still should figure out how to remove this dependency